devops/lib/component.py

```diff
@@ -92,24 +92,18 @@ def build(self, ctx: Context, dry_run=False, docker_args=None):
             logger.info(f"No Dockerfile for {self.name} component")
             return
 
-        if isinstance(docker_args, list):
-            # Insert --build-arg before each item in docker_args.
-            docker_args = [["--build-arg", docker_arg] for docker_arg in
-                           docker_args]
-            # Flatten list
-            # build_args_pair is ["--build-arg", "foo=bar"]
-            docker_args = [arg for build_args_pair in docker_args for arg in
-                           build_args_pair]
-        else:
-            docker_args = []
+        build_args = []
+        if docker_args:
+            # Insert --build-arg before each item from docker_args.
+            for docker_arg in docker_args:
+                build_args.extend(["--build-arg", docker_arg])
```

Comment:
Maybe worth thinking if `.extend([...])` is easier to read than `+= [...]`

Personally I think the `+=` is a bit easier, but `.extend` isn't all that difficult either

Reply:
For me `.extend()` is super clear in what it does, whereas `+=` I need to think if it'll do the same as `.append()` or `.extend()`. But I'm fine with either one.